packages/webapp/webapp_server.js

```diff
@@ -345,8 +345,33 @@ function getBoilerplate(request, arch) {
   return getBoilerplateAsync(request, arch).await();
 }
 
+WebApp.encodeRuntimeConfig = function (rtimeConfig) {
+  return JSON.stringify(encodeURIComponent(JSON.stringify(rtimeConfig)));
+}
+
+WebApp.decodeRuntimeConfig = function (rtimeConfig) {
+  return JSON.parse(decodeURIComponent(JSON.parse(rtimeConfig)));
+}
+
+const runtimeConfig = {
```

Comment:
A comment would be nice, explaining what hooks will contain and also what update will contain.

I would rename update to `isUpdatedByArch` or something similar

Reply:
Good point!